packages/auth-ts/package.json

```diff
@@ -8,6 +8,7 @@
   "browser": "dist/index.cjs.js",
   "module": "dist/index.esm.js",
   "esm2017": "dist/index.esm2017.js",
+  "react-native": "dist/index.rn.cjs.js",
```

Comment:
old SDK doesn't provide a react native build

Reply:
That's because the old SDK doesn't have a rollup config and doesn't support tree shaking. packages/app and packages/firebase have the react native build that the old SDK detects and abuses